doc/articles/create-an-app-vs2022.md

```diff
@@ -28,7 +28,9 @@ To create an Uno Platform app:
 
 1. Click the create button
 
-1. Wait for the projects to be created, and their dependencies to be restored
+1. Ensure that the lower left IDE icon shows a check mark and says "Ready" ![A checkmark with a text saying ready](getting-started/wizard/assets/vs2022-ready-statusbar.png). This ensures that the projects have been created, and their dependencies have been restored completely.
+
+1. A Windows Firewall window may appear. If you do not indend to do Linux with WSL development, check "Private networks" only. Otherwise, check "Public networks" and "Private networks".
```

Comment:
```suggestion
1. A Windows Firewall window may appear. If you do not intend to do Linux development with WSL, check "Private networks" only. Otherwise, check "Public networks" and "Private networks".
```